project2/index.html

```diff
@@ -0,0 +1,98 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Merriweather:400,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Noto+Sans+JP:500,700&display=swap" rel="stylesheet">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <section class="wholeting">
+        <header>
+            <h2 class="title"> Shrimp & Grits </h2>
+            <h4 class="subhead"> 30 Minute Meal </h4>
```

Comment:
Since the ingredients and directions are your `h3`s, I would consider if this should just be a `p` element or a `span` instead so it doesn't feel like you're skipping around in your hierarchy of headers.